pages/common/clear.md

```diff
@@ -2,6 +2,18 @@
 
 > Clears the screen of the terminal.
 
-- Clear the screen (equivalent to typing Control-L when using the bash shell):
+- Clear the screen (equivalent to pressing Control-L in Bash shell):
 
 `clear`
+
+- Clear the screen but keep the terminal's scrollback buffer:
+
+`clear -x`
+
+- Indicate the type of terminal to clean (normally this option is unnecessary because it is taken from the environment variable `TERM`):
+
+`clear -T {type_of_terminal}`
```

Comment:
I think you meant to have double `{{}}` here?
```suggestion
`clear -T {{type_of_terminal}}`
```

Reply:
Ups! I little slip. I will fixed it.